docs/docs/installation.mdx

```diff
@@ -65,6 +65,15 @@ A [Powershell][powershell] module is available for easy installation.
 Install-Module oh-my-posh -Scope CurrentUser -AllowPrerelease
 ```
 
+If you wish to install for all users in a linux environment, you will need to install from an elevate session
+of powershell.  You will also need to import the module one time from the elevated session in order to set the
+executable permissions correctly.
+```powershell
```

Comment:
this needs a space in between (and I need to add markdown validation)

Reply:
I think you mean a blank line between the description and the powershell code snippet?  I added a blank line there.